lib/rmt/cli/mirror.rb

```diff
@@ -1,4 +1,6 @@
 class RMT::CLI::Mirror < RMT::CLI::Base
+  class_option :do_not_raise_unpublished, desc: _('Do not fail the command if info is in alpha or beta stage'), type: :boolean, required: false
```

Comment:
> Do not fail the command if **info** is in...

What do you mean by "info"?
I would rather consider that a _product_ is in alpha or beta stage.

Reply:
Done